src/shared/utils/index.ts

```diff
@@ -13,17 +13,21 @@ const HTTPS_PORT = 51234
  * Fetches the UNL.
  *
  * @param url - The url of the validator list being fetched.
+ * @param networkId - The networkId corresponding to the url being fetched.
  * @returns A promise that resolves to a parsed UNLBlob.
  */
-export async function fetchValidatorList(url: string): Promise<UNLBlob> {
+export async function fetchValidatorList(
+  url: string,
+  networkId: string,
+): Promise<UNLBlob> {
   try {
     const resp = await axios.get(`http://${url}`)
     const unl: UNL = resp.data
     const buf = Buffer.from(unl.blob, 'base64')
     const blobParsed: UNLBlob = JSON.parse(buf.toString('ascii'))
     return blobParsed
   } catch (err: unknown) {
-    log.error('Error fetching validator List', err)
+    log.error(`Error fetching validator List for network ${networkId}`, err)
```

Comment:
Looking at this I think you could just pass in the url instead of the network id.  That would reduce passing in a variable just for error messaging.